vue/public/filing-uploader.html

```diff
@@ -7,7 +7,7 @@
   <body>
     <div id="vue-app">
       <filing-uploader
-        :form-types="[{&#39;doc_type&#39;: &#39;MC&#39;, &#39;party_code&#39;: 0}, {&#39;doc_type&#39;: &#39;AFTL&#39;, &#39;party_code&#39;: 0}, {&#39;doc_type&#39;: &#39;EFSS1&#39;, &#39;party_code&#39;: 1}, {&#39;doc_type&#39;: &#39;RDP&#39;, &#39;party_code&#39;: 0}]"
+        :form-types="[{'doc_type': 'MC', 'party_code': 0}, {'doc_type': 'AFTL', 'party_code': 0}, {'doc_type': 'EFSS1', 'party_code': 1}, {'doc_type': 'RDP', 'party_code': 0}]"
```

Comment:
@molund I changed \&#39; to single quotes for readability, but please let me know if there was a reason for using ascii code and my change breaks something.

Reply:
I don't remember a reason for the entities.  It might have been to trick a linter that wanted us to use double quotes instead of single quotes for JSON, but that's just a guess.